app/components/Review/Components.tsx

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components';
+
+export const StyledTitleRow = styled('th')`
+  padding: 16px !important;
```

Comment:
This is tech debt and was already in the codebase (I think this was me as well) but we should likely put in some css resets in the globals file so we can avoid using important which is an antipattern.

Reply:
Created #693